sdk/python/kfp/dsl/_types.py

```diff
@@ -19,72 +19,63 @@ class BaseType:
 
 # Primitive Types
 class Integer(BaseType):
-	openapi_schema_validator = {
+	openAPIV3Schema = {
```

Comment:
Nit: The name should probably be `openapiv3_schema` or `json_schema` (because this is really JsonSchema). (JsonSchema is used inside OpenAPI schema, but OpenAPI is bigger as it specifies HTTP routes etc.)